sample/core.py

```diff
@@ -161,18 +171,18 @@ def get_categoryinfo(self):
                     continue
                 if page_id not in ci_list:
                     ci_list[page_id] = []
-                ci_list[page_id] = page_content[prop]
+                ci_list[page_id] += page_content[prop]
 
         self.payload.pop('cicontinue', None)
         self.payload['prop'] = None
         return ci_list
 
-    def get_duplicatefiles(self, dflimit="max", dfdir="ascending", dflocalonly=None):
+    def get_duplicatefiles(self, dflimit="max", dfdir="ascending", dflocalonly=False):
         prop = 'duplicatefiles'
         self.payload['prop'] = prop
         self.payload['dflimit'] = dflimit
         self.payload['dfdir'] = dfdir
-        if dflocalonly is not None:
+        if dflocalonly==True:
```

Comment:
@abinashmeher999 Yes, the response of `get_categoryinfo` and `get_duplicatefiles` are dictionaries, as is seen in the log. How do we want the output?